kyuubi-server/pom.xml

```diff
@@ -410,6 +405,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.derby</groupId>
+            <artifactId>derby</artifactId>
+            <scope>test</scope>
```

Comment:
we still relies Derby to bootstrap a embedded HMS for testing Hadoop Token refresh feature.